packages/smooth_app/lib/pages/onboarding/welcome_page.dart

```diff
@@ -0,0 +1,217 @@
+import 'package:flutter/material.dart';
+import 'package:flutter/rendering.dart';
+import 'package:flutter_gen/gen_l10n/app_localizations.dart';
+import 'package:iso_countries/iso_countries.dart';
+import 'package:openfoodfacts/utils/CountryHelper.dart';
+import 'package:smooth_app/data_models/user_preferences.dart';
+import 'package:smooth_app/themes/smooth_theme.dart';
+import 'package:smooth_ui_library/util/ui_helpers.dart';
+
+/// Welcome page for first time users.
+class WelcomePage extends StatelessWidget {
+  const WelcomePage();
+
+  @override
+  Widget build(BuildContext context) {
+    final Size screenSize = MediaQuery.of(context).size;
+    // Side padding is 8% of total width.
+    final double sidePadding = screenSize.width * 8 / 100;
+    // Top padding is 16% of total width.
+    final double topPadding = screenSize.height * 16 / 100;
+    // Bottom padding is 4% of total width.
+    final double bottomPadding = screenSize.height * 4 / 100;
```

Comment:
I'm never safe with integers and division: multiplying by `.08`, `.16` or `.04` would be harmless and safer.

Reply:
no no it's not a problem, but sure, done.